src/app/CMakeLists.txt

```diff
@@ -21,13 +21,13 @@ target_link_libraries(web-eid controller ui pcsc)
 if(WIN32)
     if(CMAKE_SIZEOF_VOID_P EQUAL 8)
         set(PLATFORM x64)
-        set(SSL_PATH "C:\\OpenSSL-Win64\\bin")
     else()
         set(PLATFORM x86)
-        set(SSL_PATH "C:\\OpenSSL-Win32\\bin")
     endif()
-    if(OPENSSL_ROOT_DIR)
-        set(SSL_PATH "${OPENSSL_ROOT_DIR}\\bin")
+    if(DEFINED ENV{OPENSSL_ROOT_DIR})
+        set(SSL_PATH "$ENV{OPENSSL_ROOT_DIR}\\bin")
+    else()
+        message(FATAL_ERROR "Please set the OPENSSL_ROOT_DIR environment variable before running CMake")
```

Comment:
Why environment variable? This is built in cmake variable
https://cmake.org/cmake/help/latest/module/FindOpenSSL.html#hints

Reply:
Because the built-in variable was not available here when I tested (although `find_package(OpenSSL REQUIRED)` from `libelectronic-id` should make it transitively available as far as I understand). Should I try adding explicit `find_package(OpenSSL REQUIRED)` to this `CMakeLists.txt`?

Reply:
Fixed, adding `find_package(OpenSSL REQUIRED)` to `app/CMakeLists.txt` made `OPENSSL_ROOT_DIR` available. It actually makes sense that CMake variables don't become transitively automatically, I was mislead by `target_include_directories(PUBLIC ...)` which works transitively.

Thanks for hitting brakes on this!